src/mon/OSDMonitor.cc

```diff
@@ -154,6 +154,7 @@ void OSDMonitor::create_initial()
   newmap.set_flag(CEPH_OSDMAP_REQUIRE_LUMINOUS);
 
   newmap.full_ratio = g_conf->mon_osd_full_ratio;
+  newmap.backfill_full_ratio = g_conf->mon_osd_backfill_full_ratio;
```

Comment:
I'm just copying the initialization of these values for what I assume is the first boot.